docs/user-guide/global-configurations/deployment-window.md

```diff
@@ -0,0 +1,279 @@
+
+You can remove deployment window(s) applied to one or more apps as shown below.
+
+![Figure 18a: Removing Windows from Single App](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/remove-window-single.gif)
+
+![Figure 18b: Removing Windows from Multiple Apps](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/remove-window.gif)
+
+
+---
+
+## Checking Deployment Window
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have view-only permission or above (along with access to the environment and application) to check deployment windows.
+{% endhint %}
+
+### Overview Page
+
+The **Deployment window** section shows the deployment windows configured for each [environment](../../reference/glossary.md#environment).
+
+![Figure 19: Overview Page - Deployment Window](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/deployment-window.jpg)
+
+However, if a deployment window doesn't exist for an environment, the message `No deployment windows are configured` would be displayed next to it.
+
+You may click the dropdown icon to view the details which include:
+* Type of deployment window (Blackout/Maintenance)
+* Name and description
+* Frequency of window (once, weekly, monthly, yearly)
+* Start date and time 
+* End date and time
+
+### App Details Page
+
+Unlike the **Overview** page which shows deployment windows for all environments, the **App Details** page shows the deployment windows for one environment at a time.
+
+![Figure 20: App Details Page - Deployment Window](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/app-details-deployment-win-v2.jpg)
+
+If configured, the ongoing and upcoming deployment windows would be visible in the form of cards (in the same row that has `Application Status`, `Deployment Status`, etc.)
+
+For example, if the super-admin has configured 4 deployment windows (say 2 Blackout and 2 Maintenance), you will see 4 cards stacked upon each other. However, no cards will be shown if deployment windows aren't configured.
+
+You may click on any of them to view the details of the deployment windows.
+
+---
+
+## Result
+
+The below functions are blocked during an ongoing blackout window or outside maintenance window.
+
+* [Hibernation](#hibernation)
+* [Restart Workloads](#restart-workloads)
+* [Deletion of Workloads](#deletion-of-workloads)
+* [Deployment](#deployment)
+* [Rollback](#rollback)
+* [Deletion of CD Pipeline](#deletion-of-cd-pipeline)
+
+{% hint style="info" %}
+For exceptional cases, the exempted users specified in the deployment window configuration can perform the above actions. In case you have configured [SES or SMTP on Devtron](../global-configurations/manage-notification.md#notification-configurations), an email notification will be sent as well.
```

Comment:
It's not clear what will be the notification for

Reply:
I have removed this line now because exempted users don't seem to get any email